app/transport/http/resp.go

```diff
@@ -0,0 +1,25 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
```

Comment:
There is was a comment on the last session about renaming.
When you see `generate` prefix - you expect that something is created and returned.
It will be better to call it as `write`... xos it's main responsibility of this function.